services/m3dbnode/config/config.go

```diff
@@ -92,8 +93,11 @@ type Configuration struct {
 	// The pooling policy.
 	PoolingPolicy PoolingPolicy `yaml:"poolingPolicy"`
 
-	// The configuration for config service client.
-	ConfigService etcdclient.Configuration `yaml:"configService"`
+	// The dynamic etcd configuration for config service client.
+	ConfigService *etcdclient.Configuration `yaml:"configService"`
+
+	// The static configuration for config service client.
+	StaticConfigService *topology.StaticConfiguration `yaml:"staticConfigService"`
```

Comment:
So I think we should leave room for other static configuration in the future (not purely topology based).

Might be better to create a new struct here for that purpose, also "static" is more terse than "staticConfigService" and better describes the data described.

This could work well:

```go
StaticConfig *StaticConfiguration `yaml:"static"`
```